doc/ref/configuration/minion.rst

```diff
@@ -1181,6 +1181,17 @@ The password used for HTTP proxy access.
 
     proxy_password: obolus
 
+``no_proxy``
+------------------
+
+Default: ``[]``
+
+List of hosts to bypass proxy
```

Comment:
This should specify that it is to bypass an HTTP proxy.

Reply:
sorted